src/h5web/toolbar/HeatmapToolbar.tsx

```diff
@@ -38,6 +41,12 @@ function HeatmapToolbar(): ReactElement {
       {dataDomain && <Separator />}
 
       <ColorMapSelector value={colorMap} onChange={setColorMap} />
+      <ToggleBtn
+        label="Reverse"
+        icon={FiShuffle}
+        value={reverseColorMap}
+        onChange={toggleColorMapReversal}
+      />
```

Comment:
It would make more sense to have this `ToggleBtn` inside `ColorMapSelector` but this means another level of props to pass...